docs/bucket/notifications/README.md

```diff
@@ -407,7 +407,7 @@ Install [Redis](http://redis.io/download) server. For illustrative purposes, we
 
 This notification target supports two formats: _namespace_ and _access_.
 
-When the _namespace_ format is used, Minio synchronizes objects in the bucket with entries in a hash. For each entry, the key is formatted as "bucketName/objectName" for an object that exists in the bucket, and the value is the JSON-encoded event data about the operation that created/replaced the object in Minio. When objects are updated or deleted, the corresponding entry int he hash is updated or deleted respectively.
+When the _namespace_ format is used, Minio synchronizes objects in the bucket with entries in a hash. For each entry, the key is formatted as "bucketName/objectName" for an object that exists in the bucket, and the value is the JSON-encoded event data about the operation that created/replaced the object in Minio. When objects are updated or deleted, the corresponding entry in the hash is updated or deleted respectively.
```

Comment:
The last sentence of the paragraph can be. `When objects are updated or deleted, the corresponding entry in the hash is also updated or deleted.`